yoti-sdk-api/src/main/java/com/yoti/api/client/spi/remote/call/identity/IdentityService.java

```diff
@@ -74,9 +76,37 @@ public ShareSession createShareSession(String appId, KeyPair keyPair, ShareSessi
         }
     }
 
-    byte[] serializePayload(ShareSessionRequest shareSessionRequest) throws JsonProcessingException {
+    public ShareSessionQrCode createShareQrCode(
+            String appId,
+            KeyPair keyPair,
+            String sessionId,
+            ShareSessionQrRequest shareSessionQrRequest) {
+        notNullOrEmpty(sessionId, "Session id");
+        notNull(shareSessionQrRequest, "Share Session QR code request");
+
+        LOG.info("Requesting Share Session QR code Creation for session ID '{}", sessionId);
```

Comment:
There's a `Requesting a Share Session QR code creation...` in `YotiClient` (and logged at `debug`) that seems to be about exactly the same thing as this one, except this one is at `info` and actually has _more_ info.

You wanna avoid logging messages pertaining to the same "action/operation" that include the same info. It's fine to have them at different levels if they include different contextual information (with finer log messages including _more_ detail the the ones at coarser levels).